app/main.py

```diff
@@ -2,0 +1,14 @@
+def copy_file(command: str) -> None:
+    command_list = command.split(" ")
+    cp = command_list[0]
+    f_in = command_list[1]
```

Comment:
The variable name `f_in` is not very descriptive. Consider using a more descriptive name like `source_file`.